lib/KRadioButton.vue

```diff
@@ -220,6 +222,13 @@
          */
         this.$emit('blur');
       },
+      /**
+       * @public
+       * Set the tabIndex value
```

Comment:
Thank you for making sure its documented. Just a little improvement for clarity:

```suggestion
       * Sets `tab-index` on input
```